Decode/EventHeaderEventInfo.cs

```diff
@@ -172,5 +173,185 @@ public override string ToString()
         {
             return this.TracepointName ?? "";
         }
+
+        /// <summary>
+        /// <para>
+        /// Appends the current event identity to the provided StringBuilder as a JSON string,
+        /// e.g. <c>"MyProvider:MyEvent"</c> (including the quotation marks).
+        /// </para><para>
+        /// The event identity includes the provider name and event name, e.g. "MyProvider:MyEvent".
+        /// This is commonly used as the value of the "n" property in the JSON rendering of the event.
+        /// </para>
+        /// </summary>
+        public void AppendJsonEventIdentityTo(StringBuilder sb)
```

Comment:
Note: Method moved from enumerator. Instead of `enumerator.AppendJsonXXX()`, do `enumerator.GetEventInfo().AppendJsonXXX()`. (Usually the caller has already done a `GetEventInfo()` already.)